### ISSUE TITLE
s/HYP/FYP

### ISSUE THREAD

**Issue by lyndings** [ymichael/soc-hyp-boilerplate#4]
I'm pretty sure HYP - the term - has been deprecated? "Honours Year" doesn't apply any more since the fourth year is no longer optional.

(To be _really_ pedantic, "Final Year" isn't accurate either since you can do it slightly early too, but that _is_ the term that's currently used so myeh :p)

**Comment by shawntan** [comment]
I looked this up recently, and that module is just called "BComp Dissertation". And the terms Final and Honours vary depending which professor you ask. So "myeh".

**Comment by lyndings** [comment]
Yep, though the [SOC site](http://www.comp.nus.edu.sg/undergraduates/fyp.html) refers to it as FYP.

**Comment by ymichael** [comment]
Actually changing to FYP might make it more searchable too...

**Comment by shawntan** [comment]
fyphyp
